package.json

```diff
@@ -83,10 +83,8 @@
     "babel-preset-cozy-app": "2.0.2",
     "bundlemon": "1.3.2",
     "cozy-scripts": "8.2.0",
-    "eslint": "8.9.0",
-    "eslint-config-cozy-app": "4.0.0",
+    "eslint-config-cozy-app": "file:/home/anc/cozy/cozy-libs/packages/eslint-config-cozy-app/eslint-config-cozy-app-v7.0.0.tgz",
```

Comment:
every dependency related to eslint except the cozy config and eventual plugins that are not in the base config are deleted